Warhammer 4e Character Sheet/translation.json

```diff
@@ -1893,7 +1886,128 @@
+	"WEAPON-DWARF-PISTOL": "1H,Dwarf Pistol,Blackpowder,0,20,0,10,Blackpowder-Damaging-Impale-Penetrating-Pistol-Reload1,",
+
+	"WEAPON-ELF-BOW": "2H,Elf Bow,Bow,2,150,1,4,Damaging-Precise,",
+	"WEAPON-LONGBOW": "2H,Longbow,Bow,3,100,1,4,Damaging,",
+	"WEAPON-BOW": "2H,Bow,Bow,2,50,1,3,none,",
+	"WEAPON-SHORTBOW": "2H,Shortbow,Bow,1,20,1,2,none,",
+
+	"WEAPON-CROSSBOW-PISTOL": "1H,Crossbow Pistol,Crossbow,0,10,0,7,Pistol,",
+	"WEAPON-HEAVY-CROSSBOW": "2H,Heavy Crossbow,Crossbow,3,100,0,9,Damaging-Reload2,",
+	"WEAPON-CROSSBOW": "2H,Crossbow,Crossbow,1,60,0,9,Reload1,",
+	"WEAPON-DWARF-CROSSBOW": "2H,Dwarf Crossbow,Crossbow,1,80,0,9,Precise-Damaging-Reload 1,",
+
+	"WEAPON-REPEATER-HANDGUN": "2H,Repeater Handgun,Engineering,3,30,0,9,Blackpowder-Dangerous-Reload5-Repeater4,",
+	"WEAPON-REPEATER-PISTOL": "1H,Repeater Pistol,Engineering,1,10,0,8,Blackpowder-Dangerous-Pistol-Reload4-Repeater4,",
+	"WEAPON-PEPPERBOX": "1H,Pepperbox,Engineering,1,10,0,8,Blackpowder-Dangerous-Pistol-Reload4-Repeater4,",
+	"WEAPON-HAND-MORTAR": "2H,Hand Mortar,Engineering,3,30,0,7,Blackpowder-Dangerous-Imprecise-Reload2,",
+	"WEAPON-CANE-PISTOL": "1H,Cane Pistol,Engineering,1,10,0,8,Blackpowder-Dangerous-Imprecise-Reload6,",
+	"WEAPON-DRAKEGUN": "2H,Drakegun,Engineering,3,30,0,12,Blackpowder-Blast6-Damaging-Dangerous-Penetrating-Reload4,Inflicts 1 Ablaze on hit",
+	"WEAPON-DRAKEFIRE-PISTOL": "1H,Drakefire Pistol,Engineering,1,20,0,11,Blackpowder-Damaging-Dangerous-Impale-Penetrating-Reload4-Repeater3,Inflicts 1 Ablaze on hit",
+	"WEAPON-REPEATING-DWARF-HANDGUN": "2H,Repeating Dwarf Handgun,Engineering,3,50,0,10,Blackpowder-Dangerous-Reload5-Repeater4,",
+	"WEAPON-GRUDGE-RAKER": "2H,Grudge-raker,Engineering,2,30,0,10,Blackpowder-Dangerous-Reload5-Salvo2-Spread3,",
+
+	"WEAPON-LASSO": "1H,Lasso,Entangling,0,SB2,0,0,Entangle,",
+	"WEAPON-WHIP": "1H,Whip,Entangling,0,6,1,3,Entangle,",
+
+	"WEAPON-BOMB": "1H,Bomb,Explosives,0,SB1,0,12,Blast5-Dangerous-Impact,",
+	"WEAPON-INCENDIARY": "1H,Bow,Explosives,2,SB1,0,0,Blast4-Dangerous,Inflicts 1 Ablaze on hit",
+	"WEAPON-BLASTING-CHARGE": "1H,Blasting Charge,Explosives,0,SB1,0,12,Blast2-Dangerous-Impact-Penetrating,Inflicts 1 Ablaze on hit",
+	"WEAPON-CINDERBLAST-BOMB": "1H,Cinderblast Bomb,Explosives,0,SB2,0,10,Blast5-Dangerous-Impact-Penetrating,",
+
+	"WEAPON-SLING": "1H,Sling,Sling,0,60,0,6,none,",
+	"WEAPON-STAFF-SLING": "2H,Staff Sling,Sling,2,100,0,7,none,",
+
+	"WEAPON-BOLAS": "1H,Bolas,Throwing,0,SB3,1,0,Entangle,",
+	"WEAPON-DART": "1H,Dart,Throwing,0,SB2,1,1,Impale,",
+	"WEAPON-JAVELIN": "1H,Javelin,Throwing,1,SB3,1,3,Impale,",
+	"WEAPON-ROCK": "1H,Rock,Throwing,0,SB1,1,0,none,",
+	"WEAPON-THROWING-KNIFE": "1H,Throwing Knife,Throwing,0,SB2,1,2,none,",
+	"WEAPON-THROWING-AXE": "1H,Throwing Axe,Throwing,1,SB2,1,3,Hack,",
+	"WEAPON-DWARF-THROWING-AXE": "1H,Dwarf Throwing Axe,Throwing,1,SB2,1,4,Hack,",
+	"WEAPON-BLACKBRIAR-JAVELIN": "1H,Blackbriar Javelin,Throwing,1,SB3,1,3,Impale,Inflicts 1 Poison on hit",
```

Comment:
These can't be included. Community sheets cannot include content from the books. This aspect will need to be removed before this PR can be approved.

Reply:
@NorWhal I have removed the offending PR related content. Need me to open a new request?